rex_xai/explanation.py

```diff
@@ -307,7 +305,14 @@ def _explanation(
     exp.extract(args.strategy)
 
     if args.analyze:
-        analyze(exp, data.mode)
+        results = analyze(exp, data.mode)
+        logger.info(
```

Comment:
NB that if a user leaves the logging settings at the default level (error), they currently won't see this output! Louise and I discussed this a bit on a previous PR and decided to use `print` to output this data, to ensure that it's output if the user uses the `--analyze` regardless of their verbosity settings. What do you think?